wbooks/src/app/components/AppNavigator/config/customHeader.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Image } from 'react-native';
+import backButton from '@assets/ic_back.png';
+import headerBackground from '@assets/bc_nav_bar.png';
+import colors from '@constants/colors';
+
+import styles from '../styles';
+
```

Comment:
Wrong folder structure. Why did you create a `config` folder inside `AppNavigator`  component?

Move `wbooks/src/app/components/AppNavigator/config/customHeader.tsx` => `wbooks/src/app/components/AppNavigator/constants.tsx` or `.../config.tsx`

We only use folders for `assets`, `components` or `screens` inside other `components`/`screens`

Reply:
Changed from `AppNavigator/config/customHeader.tsx` to `AppNavigator/config.tsx`